components/actionButton/HyperActionButton.tsx

```diff
@@ -207,9 +215,10 @@ export function HyperActionButton (props: HyperActionButtonProps) {
         ]
     );
 
+
     return (
         <Button
-            css={ style ? StyleEntity.createFromDTO(style).getCssStyles() : {} }
+            css={ css }
```

Comment:
This should be something like this:

```
css={ { 
  ...(style ? StyleEntity.createFromDTO(style).getCssStyles() : {}),
  ...(css ? css : {}),
} }
```

I am not sure which one should come first though. Maybe css later since that's more specific than style.

Reply:
didn't seem to affect the styling, whichever way they were